From 1c74cdecd9ad86ade9e5ea6473b181b42ffea700 Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Wed, 3 May 2017 14:28:59 -0500
Subject: [PATCH 1002/1003] fpga: add AAL to fpga Kconfig and Makefile

Signed-off-by: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
---
 drivers/Makefile     | 1 +
 drivers/fpga/Kconfig | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/drivers/Makefile b/drivers/Makefile
index a5b9a066e994..5a39e4bde439 100644
--- a/drivers/Makefile
+++ b/drivers/Makefile
@@ -181,6 +181,7 @@ obj-$(CONFIG_STM)		+= hwtracing/stm/
 obj-$(CONFIG_ANDROID)		+= android/
 obj-$(CONFIG_NVMEM)		+= nvmem/
 obj-$(CONFIG_FPGA)		+= fpga/
+obj-$(CONFIG_AAL)		+= fpga/aal/
 obj-$(CONFIG_FSI)		+= fsi/
 obj-$(CONFIG_TEE)		+= tee/
 obj-$(CONFIG_MULTIPLEXER)	+= mux/
diff --git a/drivers/fpga/Kconfig b/drivers/fpga/Kconfig
index ad5448f718b3..fb1998091d77 100644
--- a/drivers/fpga/Kconfig
+++ b/drivers/fpga/Kconfig
@@ -117,4 +117,6 @@ config XILINX_PR_DECOUPLER
 	  region of the FPGA from the busses while that region is
 	  being reprogrammed during partial reconfig.
 
+source "drivers/fpga/aal/Kconfig"
+
 endif # FPGA
-- 
2.17.0

